src/App.tsx

```diff
@@ -2,7 +2,8 @@ import { Global } from '@emotion/core'
 import { colorsV3 } from '@hedviginsurance/brand'
 import React, { useEffect } from 'react'
 import { hot } from 'react-hot-loader'
-import { Route, Switch, useLocation } from 'react-router-dom'
+import { useLocation } from 'react-router'
```

Comment:
Ah, good point. I think this works but this is also from the current docs (they've messed around so much with these packages in the different versions i had no clue what was true anymore 😂):

> All of the components that you use in a web application should be imported from react-router-dom.

https://reactrouter.com/web/guides/primary-components

Reply:
Haha I get it! But then I guess we should maybe remove the `react-router` package altogether? 

![image](https://user-images.githubusercontent.com/42962286/112002021-c5942700-8b1f-11eb-8871-e1a710e0b25f.png)

Reply:
@styrelius Good point! I tried but we need the types from it in one place 🙄 so i think we should leave it

Reply:
Hm, maybe I'm completely "ute och cyklar" here but can't we just write that type ourselves? Nothing seems to immediately break at least 😅 
```typescript
type StaticRouterContext = {
  statusCode?: number
  url?: string
  action?: 'PUSH' | 'REPLACE'
  location?: Record<string, unknown>
}
```

Reply:
TLDR; it would work but i think its better to use the "real" types

Nah that could also work of course, but as `react-router-dom` is merely the dom bindings for `react-router` (meaning `react-router-dom` depends on `react-router` anyways so it'll need to be installed no matter if we keep it in package.json or not) i don't see why whe'd want to make it ourselves. It seems more error prone in case they would actually update something, because then we have "false" types whereas if we import it properly that would hopefully give us correct types in case they would update this type.